tools/licenses/lib/main.dart

```diff
@@ -722,72 +591,22 @@ class _RepositoryIcuLicenseFile extends _RepositoryLicenseFile {
     'ICU4C includes a configuration script generated by Autoconf,\n'
     'namely the `configure` script.)';
 
-  static String _dewrap(String s) {
-    if (!s.startsWith(' # ')) {
-      return s;
-    }
-    if (s.contains(_unexpectedHash)) {
-      throw 'ICU license file contained unexpected hash sequence';
-    }
-    if (s.contains('\x2028')) {
-      throw 'ICU license file contained unexpected line separator';
-    }
-    return s.replaceAll(_newlineHash, '\x2028').replaceAll('\n', '').replaceAll('\x2028', '\n');
-  }
-
-  static List<License> _parseLicense(fs.TextFile io) {
-    final Match? match = _pattern.firstMatch(io.readString());
-    if (match == null) {
-      throw 'could not parse ICU license file';
-    }
-    assert(match.groupCount == 14);
-    if (match.group(10)!.contains(copyrightMentionPattern) || match.group(11)!.contains('7.')) {
-      throw 'unexpected copyright in ICU license file';
-    }
-    if (!match.group(12)!.contains(gplExceptionExplanation1) || !match.group(13)!.contains(gplExceptionExplanation2)) {
-      throw 'did not find GPL exception in GPL-licensed files';
-    }
-    final List<License> result = <License>[
-      License.fromBodyAndType(_dewrap(match.group(1)!), LicenseType.unknown, origin: io.fullName),
-      License.fromBodyAndType(_dewrap(match.group(2)!), LicenseType.icu, origin: io.fullName),
-      License.fromBodyAndType(_dewrap(match.group(3)!), LicenseType.bsd, origin: io.fullName),
-      License.fromBodyAndType(_dewrap(match.group(4)!), LicenseType.bsd, origin: io.fullName),
-      License.fromBodyAndType(_dewrap(match.group(5)!), LicenseType.bsd, origin: io.fullName),
-      License.fromBodyAndType(_dewrap(match.group(6)!), LicenseType.unknown, origin: io.fullName),
-      License.fromBodyAndType(_dewrap(match.group(7)!), LicenseType.unknown, origin: io.fullName),
-      License.fromBodyAndType(_dewrap(match.group(8)!), LicenseType.bsd, origin: io.fullName),
-      License.fromBodyAndType(_dewrap(match.group(9)!), LicenseType.bsd, origin: io.fullName),
-      License.fromBodyAndType(_dewrap(match.group(11)!), LicenseType.bsd, origin: io.fullName),
-      // Matches 12 and 13 are for the GPL3 license. However, they are covered by an exemption
-      // (they are exempt because ICU4C includes a configuration script generated by Autoconf)
-      License.fromBodyAndType(_dewrap(match.group(14)!), LicenseType.mit, origin: io.fullName),
-    ];
-    return result;
+  static String _fixup(String license) {
```

Comment:
Comment about why this is needed since it seems pretty random?

Reply:
added.

and yes. ugh. that license.